src/randoop/NoExceptionCheck.java

```diff
@@ -58,7 +62,7 @@ public String toCodeStringPostStatement() {
 
   @Override
   public String toCodeStringPreStatement() {
-    return "";
+    return "// contract failure: this statement threw " + exceptionName + Globals.lineSep;
```

Comment:
The point of the comment is to provide information about what was observed during generation.

But, it will only be added because a particular exception that is tagged as an error behavior was thrown during test generation.  Any "expected" exceptions would be OK, but any invalid or error exceptions would not. We should probably wrap the statement in a try-catch.